draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1629,7 +1629,7 @@ After initialization, the `Update` function is called with constant parameters,
 
 # AEGIS as a Message Authentication Code
 
-All AEGIS variants can be used to construct a MAC.
+All AEGIS variants can be used to construct a Message Authentication Code (MAC).
```

Comment:
I'm happy for this to be capitalised because that's done in some other RFCs, like the BLAKE2/AEAD interface ones. However, we currently have this lowercase in the Introduction:

> authenticated encryption with associated data (AEAD)

Therefore, I think this should either be made lowercase or the AEAD words should be capitalised like in [RFC 5116](https://www.rfc-editor.org/rfc/rfc5116#section-1.1). Up to you.

Reply:
Authenticated Encryption with Associated Data has been capitalized.